imap/mailbox.h

```diff
@@ -383,18 +384,18 @@ typedef enum _MsgFlags {
     FLAG_DELETED            = (1<<2),
     FLAG_DRAFT              = (1<<3),
     FLAG_SEEN               = (1<<4),
+} MsgFlags;
+
+typedef enum _MsgInternalFlags {
```

Comment:
I wonder if these should be called FLAG_INTERNAL_XXX to make clear which flag goes to which bitmask. I know understand that for this particular concern, a bitfield would indeed have been less error-prone. Still, I think it's worth keeping the bitmask, but probably making clear which FLAG_ needs to go to which flag mask would bring value?

Reply:
I do agree with that. I shall update the patch to reflect that.